src/encoded/static/components/objectutils.js

```diff
@@ -618,11 +618,24 @@ export class ImageWithFallback extends React.Component {
         });
     }
 
-    // Display default "not found" image for non-existent image src
     onError() {
+        let imageAlt = null;
+        let imageUrl = null;
+
+        // IE11 has an issue where it frequently throws a "Permission denied" exception, when the image
```

Comment:
I could not figure out exactly why IE11 has the issue this comment mentions (always throws a permission exception.) So, I did the behavior in the code. IE11 has a low usage so I think it is okay. Let me know if you have an issue with it and I will think of an alternative solution